src/frontends/ir/CMakeLists.txt

```diff
@@ -4,9 +4,8 @@
 
 ov_add_frontend(NAME ir
                 FILEDESCRIPTION "FrontEnd to load OpenVINO IR file format"
-                LINK_LIBRARIES inference_engine_transformations pugixml::static
-                               # TODO: remove dependencies below in CVS-69781
-                               inference_engine inference_engine_plugin_api)
+                LINK_LIBRARIES pugixml::static
+                               openvino::runtime::dev)
```

Comment:
I suppose we need to keep comment and once it's applied, we need `openvino::runtime::dev` -> `openvino::runtime`